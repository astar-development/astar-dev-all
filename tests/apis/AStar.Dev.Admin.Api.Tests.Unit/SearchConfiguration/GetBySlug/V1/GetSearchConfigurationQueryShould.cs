using AStar.Dev.Api.Usage.Sdk.Metrics;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Admin.Api.SearchConfiguration.GetBySlug.V1;

[TestSubject(typeof(GetSearchConfigurationBySlugQuery))]
public sealed class GetSearchConfigurationQueryShould
{
    [Fact]
    public void ContainTheExpectedValues()
    {
        var sut = new GetSearchConfigurationBySlugQuery("doesn't matter", new());

        sut.ToJson().ShouldMatchApproved();
    }

    [Fact]
    public void ImplementIEndpointNameInterface()
    {
        var sut = new GetSearchConfigurationBySlugQuery("doesn't matter", new());

        sut.ShouldBeAssignableTo<IEndpointName>();
    }
}
