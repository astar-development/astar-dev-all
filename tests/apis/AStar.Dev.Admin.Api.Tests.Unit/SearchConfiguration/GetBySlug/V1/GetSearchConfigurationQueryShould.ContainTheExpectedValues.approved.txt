{
  "slug": "doesn\u0027t matter",
  "user": {
    "claims": [],
    "identities": [],
    "identity": null
  }
}